### PR TITLE
Upgrade OT api version

### PR DESCRIPTION
**Pull request by omirho**

### CODE REVIEW

**Comment by omirho** [comment]
@aabmass Can you take a look here?

Thanks!

File: nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/provider/opentelemetry.js [hunk 1]
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {context, defaultSetter} = require('@opentelemetry/api');
-const {HttpTraceContext} = require('@opentelemetry/core')
+const {context, defaultTextMapSetter} = require('@opentelemetry/api');
+const {HttpTraceContextPropagator} = require('@opentelemetry/core')
```
Comment (aabmass) [inline nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/provider/opentelemetry.js:16]: Just a heads up, this was renamed to `W3CTraceContextPropagator` in an upcoming release

File: nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/provider/opentelemetry.js [hunk 2]
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const {context, defaultSetter} = require('@opentelemetry/api');
-const {HttpTraceContext} = require('@opentelemetry/core')
+const {context, defaultTextMapSetter} = require('@opentelemetry/api');
```
Comment (aabmass) [inline nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/provider/opentelemetry.js:15]: Should `@opentelemetry/api` be in `peerDependencies` explicitly?

File: nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/test/comment.test.js [hunk 3]
```diff
@@ -197,12 +197,11 @@ describe("With OpenTelemetry tracing", () => {
 
     it('Starting an OpenTelemetry trace should produce `traceparent`', (done) => {
         const rootSpan = tracer.startSpan('rootSpan');
-
-        tracer.withSpan(rootSpan,  async () => {
+        context.with(trace.setSpan(context.active(), rootSpan), async () => {
```
Comment (aabmass) [inline nodejs/sqlcommenter-nodejs/packages/sqlcommenter-knex/test/comment.test.js:200]: Can do this simpler with [`Tracer.startActiveSpan()`](https://open-telemetry.github.io/opentelemetry-js-api/interfaces/tracer.html#startactivespan)

**Review by aabmass — APPROVED** [review]
LGTM

**Pull request merged**